ANMN/LTSP/TSaggregator/geoserverCatalog_generic.py

```diff
@@ -0,0 +1,112 @@
+## read all ANMN/ABOS files names form the geoserver
+#!/usr/bin/env python3
```

Comment:
The `#!` needs to be the _very first_ line of the file, followed by the coding. Doc strings and other comments can come after that.

Reply:
fixed